Datagrid/SimplePager.php

```diff
@@ -13,6 +13,9 @@
 
 use Doctrine\Common\Collections\ArrayCollection;
 
+/**
+ * @deprecated Will be removed in version 2.0. Use Sonata\AdminBundle\Datagrid\SimplePager instead.
+ */
```

Comment:
its hard to determine the answer of an or question with yes :-) I did it on class doc block